core/src/main/scala/com/criteo/cuttle/Database.scala

```diff
@@ -223,6 +223,14 @@ object Database {
       }
     )
   }
+
+  /* To be used only with a read-only db user as the mandatory lock check / creation is bypassed. */
+  def readOnlyConnect(dbConfig: DatabaseConfig)(implicit logger: Logger): XA = {
+    // FIXME we shouldn't use allocated as it's unsafe instead we have to flatMap on the Resource[HikariTransactor]
+    val (transactor, _) = newHikariTransactor(dbConfig).allocated.unsafeRunSync
+    logger.debug("Allocated new read-only Hikari transactor")
```

Comment:
could we change this to `logger.info` instead?

Reply:
sure!

Reply:
updated it